server/routes/appointments/create-and-edit/createRoutes.ts

```diff
@@ -94,6 +94,7 @@ export default function Create({ prisonService, activitiesService }: Services):
   get('/schedule', scheduleRoutes.GET, true)
   post('/schedule', scheduleRoutes.POST)
   get('/schedule/:prisonNumber/remove', scheduleRoutes.REMOVE, true)
+  get('/change', scheduleRoutes.CHANGE)
```

Comment:
This should be /schedule/change as it's a route within the change handler

Reply:
Actually, it routes with the schedule handler so it's OK as it is.